### ISSUE TITLE
Update `xmldoc`

### ISSUE THREAD

**Issue by grabbou** [rnpm/rnpm#156]
Waiting for release notes https://github.com/nfarina/xmldoc/issues/35

**Comment by Kureev** [comment]
Can you pick it up, Mike?

**Comment by grabbou** [comment]
Yeah, this was on hold due to lack of changelog, but since it's resolved, it's time to do it.
